docs/openstack/mariadb_copy.md

```diff
@@ -33,12 +33,12 @@ PODIFIED_MARIADB_IP=$(oc get -o yaml pod mariadb-openstack | grep podIP: | awk '
 MARIADB_IMAGE=quay.io/podified-antelope-centos9/openstack-mariadb:current-podified
 
 # Use your environment's values for these:
-EXTERNAL_MARIADB_IP=192.168.24.3
+EXTERNAL_MARIADB_IP=172.21.0.2
```

Comment:
172.21.0.100

Reply:
Actually it has to be 192.168.122.100. SSH works on the external IP too, but MariaDB only listens only on the "primary" IP of the VM.